components/formats-gpl/src/loci/formats/in/ND2Handler.java

```diff
@@ -493,7 +493,7 @@ else if (qName.equals("uiMaxDst")) {
           LOGGER.warn("Could not set the pixel type", e);
         }
       }
-      else if (qName.equals("uiWidthBytes") || qName.startsWith("uiBpc")) {
+      else if (qName.equals("uiWidthBytes") || qName.startsWith("uiBpcInMemory") || qName.startsWith("uiBpcSignificant")) {
```

Comment:
@rleigh-codelibre @dgault  did a quick audit of the metadata for the ND2  files we have in the data repository ([nd2_uiBpc.txt](https://github.com/openmicroscopy/bioformats/files/961570/nd2_uiBpc.txt)). As far as I can tell, when present `uiBpc` is either equal to `uiBpcSignificant` or lower in all cases and does not seem to be reliable for `CoreMetadata` as opposed to the two other keys.

For `uiBpcSignificant`, I think its value is already stored as `ms.bitsPerPixel` when available e.g. using [130606_JCC719_003.nd2](http://downloads.openmicroscopy.org/images/ND2/jonas/qa-7534/130606_JCC719_003.nd2), `showinf` gives:

```
Reading core metadata
filename = /ome/data_repo/curated/nd2/public/jonas/qa-7534/130606_JCC719_003.nd2
Series count = 1
Series #0 :
	Image count = 160
	RGB = false (1) 
	Interleaved = false
	Indexed = true (false color, 16-bit LUT: 3 x 65536)
	Width = 304
	Height = 232
	SizeZ = 8
	SizeT = 20
	SizeC = 1
	Thumbnail size = 128 x 97
	Endianness = intel (little)
	Dimension order = XYCZT (certain)
	Pixel type = uint16
	Valid bits per pixel = 14
	Metadata complete = true
	Thumbnail series = false
```

Should the file provided in the original issue be added to our data repository, configured and included in the daily non-regression tests?